db/test.sql

```diff
@@ -1,6 +1,6 @@
 \c nc_news_test
```

Comment:
I think you ought to gitignore this file (unless you do want a commit history of it)

Reply:
done